conf/messages.en

```diff
@@ -57,4 +57,18 @@ manufacturedPlasticWeight.details.line4 = ‘Recycled plastic’ is plastic that
 manufacturedPlasticWeight.empty.error = Enter an amount to continue
 manufacturedPlasticWeight.format.error = Amount must be entered as numbers only
 manufacturedPlasticWeight.aboveMax.error = Amount cannot exceed 99,999,999 kgs
-manufacturedPlasticWeight.invalidValue.error = Amount cannot be more than the total manufactured weight
\ No newline at end of file
+manufacturedPlasticWeight.invalidValue.error = Amount cannot be more than the total manufactured weight
+
+importedPlasticWeight.title = 2. Imported plastic packaging
```

Comment:
Just bikeshedding for a bit. What if we prefixed the returns pages with  `returns.`
Probably not needed... but it would be more organised. Although we can prefix the other ones with `account` or `home`

Just a thought.
We can discuss afterwards.

Reply:
yeap, like that, will push the change